fetch.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 mkdir -p /tmp/build
 
-curl --list-only ftp://195.255.176.166/karttainfopoiminta/ --user ${USERNAME}:${PASSWORD} > /tmp/allfiles.txt
+curl --list-only ftp://195.255.176.166/karttainfopoiminta_test/ --user ${USERNAME}:${PASSWORD} > /tmp/allfiles.txt
```

Comment:
Will be changed to an appropriate folder. Karttainfopoiminta_test contains a few light jore exports with only two lines that shouldn't take long to import.